pkgs/development/r-modules/default.nix

```
@@ -990,6 +990,10 @@ let
       preConfigure = "patchShebangs configure";
     });
 
+    RcppCGAL = old.RcppCGAL.overrideAttrs (attrs: {
+      preConfigure = "patchShebangs configure";
```

```suggestion
      postPatch = "patchShebangs configure";
```
As discussed, it's better placed in patch